README.md

```diff
@@ -54,7 +54,7 @@ English | [中文版](README_CN.md)
 
 - **Continuous Availability**
     
-    OceanBase Database adopts Paxos Consensus algorithm to achieve Zero RPO and less than 8 seconds of RTO.
+    OceanBase Database adopts Paxos Consensus algorithm to achieve Zero RPO and less than 8 seconds of RTO.Supports intra-city/remote disaster recovery, enabling multi-activity in multiple locations and zero data loss
```

Comment:
Need a space(' ') before the word 'Supports' and a dot('.') at the end of the sentence.

Reply:
thank you